pyrec/implicit/als.py

```diff
@@ -85,7 +85,14 @@ def build_index(self, index_type: str, approximate_similar_items=True, approxima
                 **kwargs
             )
             return recommender
-
+        elif index_type == 'hnsw':
+            from .hnsw import ImplicitHNSWRecommender
```

Comment:
why the `.` in `.hnsw`?

Reply:
relative path, from current folder, hnsw.py